README.md

```diff
@@ -4,7 +4,7 @@ This project was bootstrapped with [Create React App](https://github.com/faceboo
 
 In the project directory, you can run:
 
-### `yarn start`
+### `yarn run dev`
```

Comment:
this change is unnecessary to me. I've been using `npm run dev` in other projects but `npm start` is also fine. I think we don't need to change it unless we have a strong reason. It's for now consistent with backend.